### PR TITLE
Add Diamanti CSI driver

### PR DESCRIPTION
**Pull request by bcwaldon**

### CODE REVIEW

**Comment by bcwaldon** [comment]
@msau42 FYI I am on a corporate CLA. I just went through the LF account setup, so I'm not sure what needs to be done here to clear the cla/linuxfoundation check. LMK what other action needs to be taken here. Thanks!

**Comment by msau42** [comment]
Is your git commit using the same email address you used for the cla?

File: book/src/drivers.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
 [Dell EMC Unity](https://github.com/dell/csi-unity) | `csi-unity.dellemc.com` | v1.0 | A Container Storage Interface (CSI) Driver for [Dell EMC Unity](https://www.dellemc.com/en-us/storage/unity.htm) | Persistent | Read/Write Single Pod | Yes | Snapshot
 [Dell EMC VxFlexOS](https://github.com/dell/csi-vxflexos) | `csi-vxflexos.dellemc.com` | v1.0 | A Container Storage Interface (CSI) Driver for Dell EMC VxFlexOS | Persistent | Read/Write Single Pod | Yes | Snapshot
 [Dell EMC XtremIO](https://github.com/dell/csi-xtremio-deploy) | `csi-xtremio.dellemc.com` | v1.0 | A Container Storage Interface (CSI) Driver for [Dell EMC XtremIO](https://www.dellemc.com/en-us/storage/xtremio-all-flash.htm) | Persistent | Read/Write Single Pod | Yes | Snapshot
+[Diamanti](https://diamanti.com/use-cases/container-network-and-storage-new/) | `dcx.csi.diamanti.com` | v1.1 | A Container Storage Interface (CSI) Driver for Diamanti Spektra | Persistent | Read/Write Single Pod | Yes | Mirroring, Replication, Encryption, Topology Aware, Snapshot, Expansion, Raw Block
```
Comment (xing-yang) [inline book/src/drivers.md:30]: "Mirroring, Replication, Encryption" are not CSI features.  You can list them in your driver documentation, but this column is intended to show only CSI features.

**Comment by bcwaldon** [comment]
I think I've resolved the CLA issue. I'll update the content of the PR shortly.

**Comment by bcwaldon** [comment]
@msau42 @xing-yang Sorry for the noise here. Please take another look when you have a chance. Thanks!

File: book/src/drivers.md [hunk 2]
```diff
@@ -27,6 +27,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
 [Dell EMC Unity](https://github.com/dell/csi-unity) | `csi-unity.dellemc.com` | v1.0 | A Container Storage Interface (CSI) Driver for [Dell EMC Unity](https://www.dellemc.com/en-us/storage/unity.htm) | Persistent | Read/Write Single Pod | Yes | Snapshot
 [Dell EMC VxFlexOS](https://github.com/dell/csi-vxflexos) | `csi-vxflexos.dellemc.com` | v1.0 | A Container Storage Interface (CSI) Driver for Dell EMC VxFlexOS | Persistent | Read/Write Single Pod | Yes | Snapshot
 [Dell EMC XtremIO](https://github.com/dell/csi-xtremio-deploy) | `csi-xtremio.dellemc.com` | v1.0 | A Container Storage Interface (CSI) Driver for [Dell EMC XtremIO](https://www.dellemc.com/en-us/storage/xtremio-all-flash.htm) | Persistent | Read/Write Single Pod | Yes | Snapshot
+[Diamanti](https://diamanti.com/use-cases/container-network-and-storage-new/) | `dcx.csi.diamanti.com` | v1.1 | A Container Storage Interface (CSI) Driver for Diamanti Spektra | Persistent | Read/Write Single Pod | Yes | Snapshot, Expansion, Raw Block, Cloning
```
Comment (msau42) [inline book/src/drivers.md:30]: Can you reference a page that has instructions on how to install/use the csi driver?

Comment (bcwaldon) [reply book/src/drivers.md:30]: This is a proprietary driver. Is public-facing documentation/availability a requirement?

Comment (msau42) [reply book/src/drivers.md:30]: Yes, the purpose of this listing is to provide technical resources to users that are looking for how to use the driver.

Comment (bcwaldon) [reply book/src/drivers.md:30]: I see that some of the other drivers reference documentation that requires authentication. I am happy to link to docs that are on our customer-only support site. Would that be ok @msau42?

Comment (msau42) [reply book/src/drivers.md:30]: Yes that is fine. Thanks!

Comment (bcwaldon) [reply book/src/drivers.md:30]: Ok - I am going to close this PR for the time being. I will revisit in the next few weeks.

**Pull request closed without merging**